pyoos/parsers/nerrs.py

```diff
@@ -16,25 +16,26 @@ def units(param):
     # http://cdmo.baruch.sc.edu/documents/manual.pdf
     units = {
         # Water Quality
-        'Temp'      :       '°C',
```

Comment:
The fact that this was already here without the `u''` is a sign that we need more testing.